integration-tests/opentelemetry-jdbc-instrumentation/src/test/java/io/quarkus/it/opentelemetry/MariaDbLifecycleManager.java

```diff
@@ -31,6 +31,7 @@ public Map<String, String> start() {
         properties.put("quarkus.hibernate-orm.oracle.active", "false");
         properties.put("quarkus.hibernate-orm.postgresql.active", "false");
         properties.put("quarkus.hibernate-orm.db2.active", "false");
+        properties.put("quarkus.hibernate-orm.h2.active", "false");
```

Comment:
FWIW you'd be better off setting these all to `false` by default in `application.properties`, and activating only the necessary PU/datasource in the "lifecycle managers".

Reply:
Makes sense. Will do later, if you don't mind.